dydx/dydxPresenters/dydxPresenters/_v4/GlobalWorkers/Workers/dydxTransferSubaccountWorker.swift

```diff
@@ -38,6 +39,25 @@ final class dydxTransferSubaccountWorker: BaseWorker {
                 }
             }
             .store(in: &subscriptions)
+
+        AbacusStateManager.shared.state.accountBalance(of: AbacusStateManager.shared.environment?.usdcTokenInfo?.denom)
+            .withLatestFrom(
+                AbacusStateManager.shared.state.walletState
+            )
+            .sink { [weak self] balance, walletState in
+                guard let balance, balance < dydxTransferSubaccountWorker.rebalanceThreshold else { return }
+                let depositAmount = dydxTransferSubaccountWorker.rebalanceThreshold - balance
```

Comment:
To avoid excessive withdrawals, it should top-up all the way to balanceRetainAmount, ie, 

`depositAmount = balanceRetainAmount  - balance`